cmd/kubeadm/app/preflight/checks.go

```diff
@@ -148,10 +148,16 @@ func (sc ServiceCheck) Check() (warnings, errors []error) {
 				sc.Service, sc.Service))
 	}
 
-	if sc.CheckIfActive && !initSystem.ServiceIsActive(sc.Service) {
+	if sc.WantActive != initSystem.ServiceIsActive(sc.Service) {
+		status  := [2]string{"active", "not active"}
```

Comment:
I'm guessing some tests will need to be updated for this?

Reply:
```make check WHAT="./cmd/kubeadm/app/preflight" GOFLAGS="-v"``` passes with these changes. I'll need to see why the automated PR checks are failing.